web/db/characterMaterials.ts

```diff
@@ -157,3 +157,12 @@ export const RiftbornRegalia: CharacterMaterial = {
   item: registerMessage({ defaultMessage: "Riftborn Regalia" }),
   wiki: "https://genshin-impact.fandom.com/wiki/Riftborn_Regalia",
 };
+
+// possibly dropped by a new boss called Bathysmal Vishap in 2.4
+// that boss is not available in the database yet
+export const DragonheirsFalseFin: CharacterMaterial = {
```

Comment:
I noticed that Dragonheir's False Fin for Shenhe was not added in this PR, so I added it myself. My assumption is that the drop boss details are not available yet, so you chose to skip it for now. Is that correct?

Personally I would prefer the character profile to be complete, even if the materials aren't acquirable right now. With my changes, under Shenhe's profile, the item is now displayed like this:

![image](https://user-images.githubusercontent.com/27663662/144716637-68508af5-5a76-42a2-9668-5c9cdc42a744.png)

What do you think? Is this better than not showing the item at all, or would it instead cause more confusion for the users? I think it should be fine since there's a strong accuracy disclaimer at the top of the page.

Reply:
> My assumption is that the drop boss details are not available yet, so you chose to skip it for now. Is that correct?

That is correct, and I wasn't sure if leaving it out or not was preferable.

I agree it should be fine to include it given the disclaimer.

We could consider adjusting that message to make it slightly more clear though too, since I can't think of another case when an item would have this message shown. Are there any other cases? Even a small update like `This item does not currently drop from any domains.` helps suggest that it will change eventually and reduce the chance of confusion.

Reply:
> This item does not currently drop from any domains.

Sounds good!